src/data/List.js

```diff
@@ -467,6 +467,21 @@ listProto.getProvider = function () {
     return this._rawData;
 };
 
+// Get store data
+listProto.getStorage = function () {
```

Comment:
I don't see `getStorage` is used. And I think it's a good idea to expose `storage` to other modules. All data management should be in the `List` module.